snippets/src/getTransaction.ts

```diff
@@ -1,15 +1,19 @@
 import {Client} from '../../dist/npm'
+import { TransactionMetadata } from '../../src/models/common/transaction'
 
 const client = new Client('wss://s.altnet.rippletest.net:51233')
 
 getTransaction()
 
 async function getTransaction() {
   await client.connect()
-  const ledger = await client.getLedger({includeTransactions: true})
+  const ledger = await client.request({command: 'ledger', transactions: true})
   console.log(ledger)
-  const tx = await client.getTransaction(ledger.transactionHashes[0])
+  const tx = await client.request({
+    command: 'tx',
+    transaction: ledger.result.ledger.transactions[0] as string
+  })
   console.log(tx)
-  console.log('deliveredAmount:', tx.outcome.deliveredAmount)
+  console.log('deliveredAmount:', (tx.result.meta as TransactionMetadata).DeliveredAmount)
```

Comment:
Shouldn't `ledger` be of type `LedgerResponse`, and thus have metadata w/o needing the typecast?

Reply:
It's possible for `meta` to be a string, if you ask for it to be returned in binary format

Reply:
Ah gotcha, then can we check and not have to type cast.

```ts
if (typeof tx.result.meta !== 'string') {
    console.log(...)
}
```

Reply:
We know that from the request that we sent, it won't be in binary format